packages/beacon-node/src/chain/chain.ts

```diff
@@ -1223,10 +1223,15 @@ export class BeaconChain implements IBeaconChain {
           this.logger.verbose(`Updated targetCustodyGroupCount=${this.custodyConfig.targetCustodyGroupCount}`);
           this.emitter.emit(ChainEvent.updateTargetGroupCount, this.custodyConfig.targetCustodyGroupCount);
 
-          // TODO: If target group count increases, we should wait to update the advertised group until we've
+          // Need to update advertised custody prior to fulu transition so that we can serve columns we have for
+          // validators that are attached prior to the fork transition.
+          //
+          // TODO(fulu): If target group count increases, we should wait to update the advertised group until we've
           // backfilled the new groups.
-          // this.custodyConfig.updateAdvertisedCustodyGroupCount(targetCustodyGroupCount);
-          // this.emitter.emit(ChainEvent.updateAdvertisedGroupCount, this.custodyConfig.advertisedCustodyGroupCount);
+          if (this.config.getForkSeq(slot) < ForkSeq.fulu) {
```

Comment:
need to confirm `FULU_FORK_EPOCH` is defined first?

Reply:
If we have `ForkSeq.fulu` that implies `FULU_FORK_EPOCH` is defined. It can be `Infinity` though. In that case I think the logic still valid as the current fork will be behind fulu.

Reply:
We discussed this at interop. I will add a charge to the spec. It needs to be advertised beforehand for peer management at the transition

Reply:
>  It needs to be advertised beforehand for peer management at the transition

 but still the case as soon as we set the fork epoch value, in practice this will likely 2-3 weeks before the fork happens